src/elements/content-preview/ContentPreview.js

```diff
@@ -1274,6 +1283,7 @@ class ContentPreview extends React.PureComponent<Props, State> {
                             onDownload={this.download}
                             contentOpenWithProps={contentOpenWithProps}
                             canAnnotate={this.canAnnotate()}
+                            previewExperiences={previewExperiences}
```

Comment:
This version of the preview header is only shown to platform customers, so this may not be needed.

Reply:
yes, I will remove, thanks!